Gems/PhysX/Code/Source/HingeJointComponent.cpp

```diff
@@ -85,4 +88,60 @@ namespace PhysX
             m_jointSceneOwner = leadFollowerInfo.m_followerBody->m_sceneOwner;
         }
     }
+
+    // AZ::Component
+    void HingeJointComponent::Activate() {
+        JointComponent::Activate();
+        PhysX::JointInterfaceRequestBus::Handler::BusConnect(this->GetEntityId());
+    }
+
+    void HingeJointComponent::Deactivate() {
+        JointComponent::Deactivate();
+        PhysX::JointInterfaceRequestBus::Handler::BusDisconnect();
+    }
```

Comment:
nit: Braces again (clang-format should fix this)

Also it's usually good practice to disconnect and deactivate in the reverse order, so this would be...

```
PhysX::JointInterfaceRequestBus::Handler::BusDisconnect();
JointComponent::Deactivate();
```

instead, thanks!

Reply:
Adjusted order.